mobile/components/header.js

```diff
@@ -0,0 +1,31 @@
+import { View, Image, Pressable, Text } from "react-native";
+import { styles } from "../styles/basic_styles";
+import { FontAwesome5 } from "@expo/vector-icons";
+import { useNavigation } from "@react-navigation/native";
+import Line from "../components/line";
+
+export default function Header(props) {
+    const nav = useNavigation();
```

Comment:
Er moet op de header een max -width op dit moment valt de terug pijl buiten het scherm als de titel te lang is.